packages/next/server/dev/next-dev-server.ts

```diff
@@ -831,17 +831,23 @@ export default class DevServer extends Server {
             const { originalCodeFrame, originalStackFrame } = originalFrame
             const { file, lineNumber, column, methodName } = originalStackFrame
 
-            console.error(
-              (type === 'warning' ? chalk.yellow('warn') : chalk.red('error')) +
-                ' - ' +
-                `${file} (${lineNumber}:${column}) @ ${methodName}`
+            Log[type === 'warning' ? 'warn' : 'error'](
+              `${file} (${lineNumber}:${column}) @ ${methodName}`
             )
-            console.error(
-              `${(type === 'warning' ? chalk.yellow : chalk.red)(err.name)}: ${
-                err.message
-              }`
-            )
-            console.error(originalCodeFrame)
+            if (src === 'edge-server') {
```

Comment:
Middleware errors are different because of `function withTaggedErrors(fn: RunnerFn)`. Need this check to not break `middleware-dev-errors` tests..